src/XMLimport.h

```diff
@@ -110,23 +110,23 @@ class XMLimport : public QXmlStreamReader
 
     QPointer<Host> mpHost;
     QString mPackageName;
-    TTrigger* mpTrigger;
-    TTimer* mpTimer;
-    TAlias* mpAlias;
-    TKey* mpKey;
-    TAction* mpAction;
-    TScript* mpScript;
-    TVar* mpVar;
-    bool gotTrigger;
-    bool gotTimer;
-    bool gotAlias;
-    bool gotKey;
-    bool gotAction;
-    bool gotScript;
-    int module;
-    int mMaxRoomId;
-    quint8 mVersionMajor;
-    quint16 mVersionMinor; // Cannot be a quint8 as that only allows x.255 for the decimal
+    TTrigger* mpTrigger = nullptr;
+    TTimer* mpTimer = nullptr;
+    TAlias* mpAlias = nullptr;
+    TKey* mpKey = nullptr;
+    TAction* mpAction = nullptr;
+    TScript* mpScript = nullptr;
+    TVar* mpVar = nullptr;
+    bool gotTrigger = false;
+    bool gotTimer = false;
+    bool gotAlias = false;
+    bool gotKey = false;
+    bool gotAction = false;
+    bool gotScript = false;
+    int module = 0;
+    int mMaxRoomId = 0;
+    quint8 mVersionMajor = 1;
+    quint16 mVersionMinor = 0; // Cannot be a quint8 as that only allows x.255 for the decimal
```

Comment:
Keep comments from cpp file
```suggestion
    quint8 mVersionMajor = 1; // 0 to 255
    quint16 mVersionMinor = 0; // 0 to 999 for 3 digit decimal value. Cannot be a quint8 as that only allows x.255 for the decimal
```